tools/schemacode/versioneer.py

```diff
@@ -0,0 +1,1886 @@
+# Version: 0.18
```

Comment:
why not use version 0.21?

https://github.com/python-versioneer/python-versioneer/tags

Reply:
Good idea. I updated it in 31bcd196b601481b73c5c6f1101d9f60cdb37fc9.